src/storage-controller/src/lib.rs

```diff
@@ -321,6 +317,7 @@ where
             // CollectionManager when they are created. We only bring ourselves
             // up to date with collection contents or do any preparatory
             // consolidation work when we actually start writing to them!
+            // TODO(jkosh44) This no longer works.
```

Comment:
@aljoscha Is it safe to remove this entire method? From what I can tell it is being used in a couple of places.

Reply:
Never mind, I see that https://github.com/MaterializeInc/materialize/pull/29699 deletes all of this code and all of it's callers. I'll probably wait for that PR to merge and then rebase this one.